Using IndexedDB in Blazor WASM App/Data/ExampleDb.cs

```diff
@@ -4,18 +4,18 @@
 
 namespace Blazor_WebAssembly_App.Data
 {
-    public class ExampleDb : IndexedDb
+    public class IndexDb : IndexedDb
     {
-        public ExampleDb ( IJSRuntime jSRuntime, string name, int version ) : base(jSRuntime, name, version) { }
+        public IndexDb ( IJSRuntime jSRuntime, string name, int version ) : base(jSRuntime, name, version) { }
 
         // These are like tables. Declare as many of them as you want.
-        public IndexedSet<Person> People { get; set; }
+        public IndexedSet<Employee> employee { get; set; }

```

Comment:
Naming convention should follow.

employee => Employees

Reply:
In this case, may I use the singular form, i.e., 'Employee'? Singular form class names are commonly used in Microsoft documentation.